### PR TITLE
Set error when using invalid IDs in history version comparison

### LINKED ISSUE #29678: [com_contenthistory] Multiple unhandled PHP notices when ids dont exist
### Steps to reproduce the issue

af6eddfa0a3bd1d85c9964c25d21b706c64993a4

Go to faked admin url: 
http://127.0.0.1/administrator/index.php?option=com_contenthistory&view=compare&layout=compare&tmpl=component&cb519da4124ac219494e83630c56cb09=1&id1=&id2=

Screenshot 2020-06-17 at 12 10 28

or http://127.0.0.1/administrator/index.php?option=com_contenthistory&view=compare&layout=compare&tmpl=component&cb519da4124ac219494e83630c56cb09=1&id1=1&id2=

Screenshot 2020-06-17 at 12 10 46

### PR DESCRIPTION
**Pull request by SharkyKZ**
Fixes https://github.com/joomla/joomla-cms/issues/29678.

### Summary of Changes

Fixes notices when using invalid IDs in history version comparison.

### Testing Instructions

Go to faked admin url (change `cb519da4124ac219494e83630c56cb09` to your session token):
http://127.0.0.1/administrator/index.php?option=com_contenthistory&view=compare&layout=compare&tmpl=component&cb519da4124ac219494e83630c56cb09=1&id1=&id2=

### Expected result

No notices.

### Actual result

Notices like these:

>Notice: Undefined property: stdClass::$id in administrator\components\com_contenthistory\views\compare\tmpl\compare.php on line 63
>Notice: Trying to get property 'value' of non-object in administrator\components\com_contenthistory\views\compare\tmpl\compare.php on line 63

### Documentation Changes Required

No.

### CODE REVIEW

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on f31bfa286ac700f95e0e01ba04698d604be9993f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29683">issues.joomla.org/tracker/joomla-cms/29683</a>.</sub>

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on f31bfa286ac700f95e0e01ba04698d604be9993f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29683">issues.joomla.org/tracker/joomla-cms/29683</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29683">issues.joomla.org/tracker/joomla-cms/29683</a>.</sub>

File: administrator/language/en-GB/en-GB.com_contenthistory.ini [hunk 1]
```diff
@@ -31,6 +31,9 @@ COM_CONTENTHISTORY_COMPARE_TITLE="Compare View"
 COM_CONTENTHISTORY_COMPARE_VALUE1="Saved on %s %s"
 COM_CONTENTHISTORY_COMPARE_VALUE2="Saved on %s %s"
 COM_CONTENTHISTORY_ERROR_KEEP_NOT_PERMITTED="You are not permitted to change the keep forever status."
+COM_CONTENTHISTORY_ERROR_FAILED_LOADING_CONTENT_TYPE="Failed loading content type."
+COM_CONTENTHISTORY_ERROR_INVALID_ID="Invalid ID selected."
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.com_contenthistory.ini]:
```suggestion
COM_CONTENTHISTORY_ERROR_FAILED_LOADING_CONTENT_TYPE="Failed loading content type."
COM_CONTENTHISTORY_ERROR_INVALID_ID="Invalid ID selected."
COM_CONTENTHISTORY_ERROR_KEEP_NOT_PERMITTED="You are not permitted to change the keep forever status."
```

**Comment by richard67** [comment]
Adding back the test results because last change after tests was just language string reordering.

RTC also still valid.

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on e491f0d58b4db7e412c5c61df849cd60b9edbe08<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29683">issues.joomla.org/tracker/joomla-cms/29683</a>.</sub>

**Comment by zero-24** [comment]
Merging

**Pull request merged**